### ISSUE TITLE
Error 500 after migrating github repository via api

### ISSUE THREAD

**Issue by freebackup** [go-gitea/gitea#8607]
- Gitea version (or commit ref):  
1.11.0+dev-56-g28f60bb5c built with GNU Make 4.2.1, go1.13.3 : bindata, sqlite, sqlite_unlock_notify
- Git version:  N/A
- Operating system:   docker on ubuntu
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [X] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [X] Not relevant
- Log gist:

## Description

I am attempting to migrate github repositories to gitea.  Each migration appears to work, but the repository is inaccessible in the GUI (display an Error 500 page).

```
$data='{
  "auth_username": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
  "clone_addr": "https://github.com/user/repository",
  "private": true,
  "mirror": true,
  "uid": 3,
  "repo_name": "repository",
  "auth_password": "",
  "description": ""
}'

$ curl -kvs -H 'Content-Type: application/json' -XPOST -d "$data" -H 'Authorization: token xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx' 'http://gitea:3000/api/v1/repos/migrate

{
  "id": 19,
  "owner": {
    "id": 3,
    "login": "myuser",
    "full_name": "",
    "email": "",
    "avatar_url": "https://gitea.local/user/avatar/myuser/-1",
    "language": "",
    "is_admin": false,
    "last_login": "1970-01-01T00:00:00Z",
    "created": "2019-05-28T01:59:28Z",
    "username": "myuser"
  },
  "name": "repository",
  "full_name": "myuser/repository",
  "description": "Common Place to put documentation about VISION",
  "empty": false,
  "private": true,
  "fork": false,
  "parent": null,
  "mirror": true,
  "size": 1063,
  "html_url": "https://gitea.local/myuser/repository",
  "ssh_url": "git@gitea.local:myuser/repository.git",
  "clone_url": "https://gitea.local/myuser/repository.git",
  "original_url": "",
  "website": "",
  "stars_count": 0,
  "forks_count": 0,
  "watchers_count": 1,
  "open_issues_count": 0,
  "default_branch": "master",
  "archived": false,
  "created_at": "2019-10-21T04:17:17Z",
  "updated_at": "2019-10-21T04:17:21Z",
  "permissions": {
    "admin": true,
    "push": true,
    "pull": true
  },
  "has_issues": true,
  "internal_tracker": {
    "enable_time_tracker": true,
    "allow_only_contributors_to_track_time": true,
    "enable_issue_dependencies": true
  },
  "has_wiki": true,
  "has_pull_requests": true,
  "ignore_whitespace_conflicts": false,
  "allow_merge_commits": true,
  "allow_rebase": true,
  "allow_rebase_explicit": true,
  "allow_squash_merge": true,
  "avatar_url": ""
}
```

At this point the repository exists, but returns Error 500 when you attempt to browse to it.

I discovered after a while that the following command;

```
sqlite gitea.db 'update repository set status=0'
```

would immediately unlock all the repositories, and I could browse and interact with them.

These are NOT large repositories, most between 1 and 25 megabytes.

Am I doing something wrong? Not waiting long enough? Or is something broken?

Thanks
Dave

**Comment by systemmonkey42** [comment]
I installed a fresh instance (docker gitea/gitea:latest), and the error still occurred.

I did another fresh install with an older image - Gitea Version: 1.8.3+2-g11f6ed4f8   (via docker image gitea/gitea:1.8) and the problem didn't occur, so I suspect its an issue with the latest release.

**Comment by afocus** [comment]
me too

**Comment by max-wittig** [comment]
Same here.

**Comment by max-wittig** [comment]
Downgrading to 1.9.4 is working. I will play around with 1.10 again, when this issue is fixed.

**Comment by yukunyi** [comment]
1.13.3  has same error.
the command is work: `sqlite gitea.db 'update repository set status=0'`

**Comment by davidlt** [comment]
The issue is still present int 1.10.1. IIUC this issue should be solved in 1.11.0 in 2020 Jan.

**Comment by lunny** [comment]
Maybe another of my PR on v1.10.2 fixed this? #9511

**Comment by lunny** [comment]
@6543 Maybe @davidlt could confirm that.
